elastic-job-lite-console/src/main/java/org/apache/shardingsphere/elasticjob/lite/console/dto/request/FindJobExecutionEventsRequest.java

```diff
@@ -55,8 +55,8 @@
      * @param pageSize page size
      */
     public FindJobExecutionEventsRequest(final Integer pageSize, final Integer pageNumber) {
-        this.pageSize = pageSize;
-        this.pageNumber = pageNumber;
+        setPageSize(pageSize);
```

Comment:
Please just use this.xxx to assign regular fields

Reply:
done